llvm/test/Transforms/DCE/intrinsics-willreturn.ll

```diff
@@ -0,0 +1,1646 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S < %s -passes=dce  | FileCheck %s
+
+
+; BPF
+
+declare i64 @llvm.bpf.load.half(ptr, i64)
+declare i64 @llvm.bpf.load.word(ptr, i64)
+declare i64 @llvm.bpf.load.byte(ptr, i64)
```

Comment:
Split this into separate files for separate targets. there's been a long term desire to not have intrinsics included if the target isn't built, but this works today

Reply:
Done